librz/core/cmd/cmd_alias.c

```diff
@@ -117,9 +117,9 @@ RZ_IPI RzCmdStatus rz_alias_handler(RzCore *core, int argc, const char **argv) {
 				rz_core_cmd0(core, v);
 			}
 		} else {
-			ut64 at = rz_num_get(core->num, buf + 1);
-			if (at != UT64_MAX) {
-				rz_core_seek(core, at, true);
+			RzFlagItem *flag = rz_flag_get(core->flags, buf + 1);
+			if (flag) {
+				rz_core_seek(core, flag->offset, true);
```

Comment:
what is the purpose of this whole piece of code?

Reply:
![$-flag](https://github.com/rizinorg/rizin/assets/12002672/78f52e96-b944-42da-8084-e099f0b230dd)
(from `$??`)